contracts/TokenInterface.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
```

Comment:
Cant you just import OZ ERC20 contract?

Reply:
Yes, need to do that for the interface. Working on it.